### PR TITLE
fix: #1841 popover refactoring 

### LINKED ISSUE #1841: Popover Fundamental-styles 0.5.0 adoption
Popover component has been refactored in version 0.5.0 of Fundamental-styles. Fundamental-ngx needs to adopt these changes.

### PR DESCRIPTION
**Pull request by InnaAtanasova**
#### Please provide a link to the associated issue.
closes #1841
#### Please provide a brief summary of this pull request.
Added Header, Subheader and Footer directives
#### Please check whether the PR fulfills the following requirements

- [x] the commit message follows the guideline:
https://github.com/SAP/fundamental-ngx/blob/master/CONTRIBUTING.md
- [x] tests for the changes that have been done

Documentation checklist:
- [x] update `README.md`
- [x] [Breaking Changes Wiki](https://github.com/SAP/fundamental-ngx/wiki/Breaking-Changes)
- [x] Documentation Examples
- [x] Stackblitz works for all examples

### CODE REVIEW

File: libs/core/src/lib/toggle/toggle.component.scss [hunk 1]
```diff
@@ -1,4 +1,4 @@
-@import "~fundamental-styles/dist/toggle";
+@import "~fundamental-styles/dist/switch";
```
Comment (stefanoScalzo) [inline libs/core/src/lib/toggle/toggle.component.scss]: should this be in this PR?

**Comment by mikerodonnell89** [comment]
Popover docs page is throwing `ExpressionChangedAfterItHasBeenCheckedError`

File: libs/core/src/lib/toggle/toggle.component.scss (hunk 1 above)
Comment (InnaAtanasova) [reply libs/core/src/lib/toggle/toggle.component.scss]: yes

File: libs/core/src/lib/popover/popover-body/popover-body-directives/popover-body-footer.directive.ts [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import { Directive, HostBinding, Input } from '@angular/core';
+
+@Directive({
+    selector: '[fdPopoverBodyFooter], [fd-popover-body-footer]'
+})
+export class PopoverBodyFooterDirective {
+    /** @hidden */
+    @HostBinding('class.fd-popover__body-footer')
```
Comment (salarenko) [inline libs/core/src/lib/popover/popover-body/popover-body-directives/popover-body-footer.directive.ts]: I recommend to use `readonly` for such case.

**Comment by InnaAtanasova** [comment]
> Popover docs page is throwing `ExpressionChangedAfterItHasBeenCheckedError`

It's been fixed

**Pull request merged**